source/sio/ip/resolve.hpp

```diff
@@ -340,8 +340,8 @@ namespace sio::async {
       ip::resolver_query query_;
 
       template <decays_to<sender> Self, class Receiver>
-      static operation<Scheduler, Receiver>
-        subscribe(Self&& self, exec::subscribe_t, Receiver&& receiver) {
+      friend operation<Scheduler, Receiver>
+        tag_invoke(exec::subscribe_t, Self&& self, Receiver receiver) {
```

Comment:
changing the sequence related cpos to member cpos need changes in stdexec, right?

Reply:
Sure, we should adjust stdexec first.